### PR TITLE
Change in LP1

### PR DESCRIPTION
**Pull request by bhoghem**

### CODE REVIEW

**Comment by bhoghem** [comment]
Input sudah terbaru

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -236,7 +236,7 @@
     </androidx.cardview.widget.CardView>
 
     <TextView
-        android:id="@+id/textView2"
+        android:id="@+id/tv_input_name2"
```
Comment (meirusfandi) [inline app/src/main/res/layout/activity_main.xml]: change with **tv_input_name_player** or **tv_input_name_opponent**

File: app/src/main/res/layout/activity_main2.xml [hunk 2]
```diff
@@ -236,7 +236,7 @@
     </androidx.cardview.widget.CardView>
 
     <TextView
-        android:id="@+id/textView2"
+        android:id="@+id/tv_input_name2"
```
Comment (meirusfandi) [inline app/src/main/res/layout/activity_main2.xml]: change with **tv_input_name_player** or **tv_input_name_opponent**

**Review by meirusfandi — COMMENTED** [review]
update id to specific function

**Pull request merged**